website_product_brand/controllers/main.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2012-Today Serpent Consulting Services Pvt. Ltd.
+#                                     (<http://www.serpentcs.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>
+#
+##############################################################################
+
+import werkzeug
+from openerp import http
+from openerp.http import request
+import openerp.addons.website_sale.controllers.main
+from openerp import SUPERUSER_ID
+from openerp.addons.website.models.website import slug
+from openerp.addons.website_sale.controllers.main import table_compute
+from openerp.addons.website_sale.controllers.main import QueryURL
```

Comment:
Group and order imports according to [the guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports)